src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -14,37 +14,55 @@ redirects:
   - /docs/accounts/accounts-billing/new-relic-one-pricing-billing
   - /docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions
   - /docs/licenses/license-information/faq/frequently-asked-questions
+  - /docs/accounts/accounts-billing/new-relic-one-pricing-billing/billing-ui
+  - /docs/accounts/accounts-billing/general-account-settings/introduction-account-settings
 ---
 
-New Relic has two primary billing factors: ingested data and billable users. In this doc, we'll explain how pricing works, what you get for free, and other details. 
+import accountsUsageUi from 'images/accounts_screenshot-full_usage-ui.webp'
+
+The New Relic pricing model is simple and relies on two primary billing factors: how much data you ingest and how many billable users you have. We'll explain where to find your billing UI, how our pricing plan works, what you get for free, and other details. 
 
 <Callout variant="tip">
-This doc is about our newer usage-based pricing model, and not our original pricing model. You're probably on our newer pricing model, but if you're not sure, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
+A small number of older New Relic customers are on an older pricing model. For docs on that, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
```

Comment:
"Docs on that" seems a little informal. Maybe just "for information" or some variant thereof?